src/collector.rs

```diff
@@ -61,79 +61,79 @@ struct RecorderHandle(mpsc::Sender<Datum>);
 
 struct Collector {
     metrics_data: BTreeMap<Timestamp, HashMap<(Key, Kind), StatisticSet>>,
-    config: Config,
+    config: &'static Config,
```

Comment:
Why change this to a reference?

Reply:
To get rid of arbitrary cloning here and there - the Config object might as well be a static ref I thought.

Reply:
Ok, though it affected the public API, but it seems not to.